code_samples/paging_predicate_sample/Comparator.java

```diff
@@ -0,0 +1,46 @@
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class Comparator implements java.util.Comparator<Map.Entry<String, Double>>, IdentifiedDataSerializable {
+    public static final int CLASS_ID = 103;
+
+    public Comparator() {
+    }
+
+    @Override
+    public void writeData(ObjectDataOutput out) throws IOException {
+    }
+
+    @Override
+    public void readData(ObjectDataInput in) throws IOException {
+    }
+
+    // We needed a Double comparator since the default number type is 'double' in Node.js client. You can change it via defaultNumberType serialization setting.
```

Comment:
```suggestion
    // We needed a Double comparator since the default number type is 'double' in Node.js client. 
    // You can change it via defaultNumberType serialization setting.
```
To make it more readable